bowtie/_cli.py

```diff
@@ -1287,7 +1297,19 @@ async def _run(
                 seq_case = SeqCase(seq=count, case=case)
                 case_reporter = reporter.case_started(seq_case)
 
+                if (
+                    "$schema" in seq_case.case.schema
+                    and _validate_schema_and_dialect(
```

Comment:
After you fix your issue above and simply do `foo == bar`, it's literally shorter to inline the one line here than it is to define this function :)